tools/ci/flink-ci-tools/src/main/java/org/apache/flink/tools/ci/licensecheck/JarFileChecker.java

```diff
@@ -210,11 +210,6 @@ private static int findNonBinaryFilesContainingText(
                     // dual-licensed under GPL 2 and EPL 2.0
                     // contained in sql-avro-confluent-registry
                     .filter(path -> !pathStartsWith(path, "/org/glassfish/jersey/internal"))
-                    // contained in sql-connector-pulsar
-                    .filter(
-                            path ->
-                                    !pathStartsWith(
-                                            path, "/org/apache/pulsar/shade/org/glassfish/jersey/"))
```

Comment:
We need to keep this for now, as otherwise CI will fail in the pulsar repo (eventually).